src/elements/walletConnect/WalletConnectModal.tsx

```diff
@@ -0,0 +1,29 @@
+import { Modal } from 'components/modal/Modal';
+import { UseWalletConnect } from './useWalletConnect';
+import { WalletConnectModalList } from './WalletConnectModalList';
+import { WalletConnectModalPending } from './WalletConnectModalPending';
+import { WalletConnectModalError } from './WalletConnectModalError';
+
+export const WalletConnectModal = (props: UseWalletConnect) => {
+  const { isPending, isError } = props;
+
+  const title = isError
```

Comment:
Best to either useMemo or take this function outside

Reply:
done